src/scenes/shop.js

```diff
@@ -182,7 +183,7 @@ export class ShopScene extends Phaser.Scene {
             goldText = gold / 1000 + "k";
         } else if (gold > 10000000) {
             goldText = gold / 1000000 + "M";
-            fill = "#06c663";
+            color = "#06c663";
```

Comment:
Unrelated, but could we move the cash stack location? Kinda looks like the Bot price. Maybe the bottom right corner?

Reply:
I'll test it out later and see if I can find somewhere it looks good

Reply:
Added a trello story